fusedrug/data/tokenizer/modulartokenizer/scripts/add_tokenizer_to_multi_tokenizer.py

```diff
@@ -38,16 +28,7 @@ def main(cfg: DictConfig) -> None:
 
     t_mult = ModularTokenizer.load(path=cfg_raw["data"]["tokenizer"]["in_path"])
 
-    test_tokenizer(t_mult, cfg_raw=cfg_raw, mode="loaded_path")
```

Comment:
This is a sanity check for the tokenizer, intended to catch problems before they hit hard.  Why remove this?

Reply:
Cause it's should be a generic test here, and this specific test is not in the right place.
Foe example, the script will not necessarily add a subtokenizer to modular tokenizer that already contains AA subtokenizer.